Fix:

scamper_control.c:3801: error: ‘AI_NUMERICSERV’ undeclared (first use in this function)

From: https://github.com/macports/macports-legacy-support/blob/v1.3.0/include/netdb.h#L23
===================================================================
--- scamper/scamper_control.c	2024-08-17 22:11:05.000000000 +0000
+++ scamper/scamper_control.c	2024-12-14 05:01:54.867327356 +0000
@@ -65,6 +65,10 @@
 #include "host/scamper_host_do.h"
 #endif
 
+#ifndef AI_NUMERICSERV
+#define AI_NUMERICSERV 0x00001000
+#endif
+
 #define REMOTE_HDRLEN 10
 
 /*
